modules/ROOT/pages/authentication-authorization/privileges-and-roles-immutable.adoc

```diff
@@ -0,0 +1,150 @@
+Instead, they can only be administered when auth is disabled -- that is, when the configuration setting xref:configuration/configuration-settings.adoc#config_dbms.security.auth_enabled[`dbms.security.auth_enabled`] is set to `false`.
+
+[CAUTION]
+Immutable privileges and roles should only be used in a scenario where changes are rare. 
+They are intentionally difficult to change and thus changes should be undertaken with caution (i.e. when the DBMS has been isolated by some other means and unauthorized access can be reliably prevented).
+It is considered to be the kind of action which may be performed once during the commissioning phase of a DBMS.
+
+When the configuration setting xref:configuration/configuration-settings.adoc#config_dbms.security.auth_enabled[`dbms.security.auth_enabled`] is set to `false`, immutable privileges and roles can be administered in a similar way to regular privileges and roles, using the `IMMUTABLE` keyword.
+
+Once the dbms is safely isolated from external connections, follow these steps to administer immutable privileges and roles:
+
+. Change the config setting xref:configuration/configuration-settings.adoc#config_dbms.security.auth_enabled[`dbms.security.auth_enabled`] to `false`.
+. Restart the DBMS.
+. Create or remove immutable privileges and roles in the same way as regular privileges and roles but with the addition of the `IMMUTABLE` keyword
+(see xref:authentication-authorization/privileges-and-roles-immutable.adoc#access-control-privileges-immutable-usecase[here] for examples).
+. Change the config setting xref:configuration/configuration-settings.adoc#config_dbms.security.auth_enabled[`dbms.security.auth_enabled`] back to `true`.
+. Restart the DBMS.
+
+Privileges and roles created in this way will now appear to all users as an immutable part of the DBMS.
+The only way to subsequently change or remove them it would be to repeat the process of setting xref:configuration/configuration-settings.adoc#config_dbms.security.auth_enabled[`dbms.security.auth_enabled`] to `false`.
+
+
+[[access-control-privileges-immutable-usecase]]
+== When to use immutable privileges and roles
+
+Immutable privileges and roles are useful for restricting the actions of users who can themselves xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-privilege-management[administer privileges] and xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-role-management[roles].
+
+=== Immutable privileges
+
+You may want to prevent all users from performing Database Management, even the `admin` user (who are themselves able to add or remove privileges).
```

Comment:
I find this section overly explanatory. Instead of telling them how to prevent users from performing database management, we first tell them what not to do and then give them an example of using immutable privileges. Would it be more helpful if you directly told them how to do it?

Reply:
yes good point. Will re-phrase

Reply:
I'll do it as part of my review commit —no problem. I just wondered if there is any particular reason for explaining to the users the wrong way of doing it before telling them how to do it.